_posts/2024-08-27-updating-member-details.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  A better way to update union member details
+description:
+  Making it easy for members to update their details can result in better retention, more revenue and happier members. Here's how Join Together can help you do this.
+og_image_path: 
+date:   2024-08-27 11:19:13 +0000
+---
+
+We’ve worked with unions for years, and heard all sorts of horror stories about the difficulties of staying in touch with members. To try and fix this, we’ve seen unions taking a variety of approaches. These include:
```

Comment:
Are there really "Horror stories" about "...staying in touch with members"? If there are, should we say what they are? Or are the horror stories about trying to keep member data up to date?